charts/astronomer/templates/registry/registry-configmap.yaml

```diff
@@ -34,6 +34,10 @@ data:
     {{- else }}
       filesystem:
         rootdirectory: /var/lib/registry
+    {{- end }}
+    {{- if (.Values.registry.redirect).disable }}
```

Comment:
Why do you use `()` here ?

Also, I can see in many places in the Chart we have `feature.enabled`, I think it would be more consistent to change .disable to .enabled and set default to true